### ISSUE TITLE
Color commenting is not working as intended

### ISSUE THREAD

**Issue by mcyc** [AASJournals/AASTeX60#101]
The color commenting commands, at least for AASTeX v6.3, do not change the text color when the `trackchanges` option is on, as suggested in the [documentation](https://journals.aas.org/aastexguide/#revised_color). Similarly, `\edit1{<text>}`, `\edit2{<text>}` and `\edit3{<text>}`, commands do not resulted in the advertised color changes either.

**Comment by augustfly** [comment]
Thanks. A minimum working example would always be appreciated, but I came up with this one that confirms that there is no color in any of the v6.3 markup tools (at least on my computer too). I'm marking this as a pretty serious bug. @gregschwarz ??

```
\documentclass[twocolumn,linenumbers,longauthor,trackchanges]{aastex63}

\shorttitle{text}
\shortauthors{text}

\begin{document}

\title{testing edits}
\author[0000-0012-3245-1234]{Author Name}
\affiliation{Kansas Technology College}


\begin{abstract}
abstract text
\end{abstract}


\section{testing}

this is a block of text. i want to \added{add more text to it now.}
 
\edit1{Some edits} are have been made.

More \edit2{edits happened recently and again} later.

Someday I'll finish \edit3{all my edits}.

This is the end of the document. I really should \deleted{delete all this text.}

\authorcomment1{I think this should be gold OA.}

\authorcomment2{I think Bob and Jill should be co-authors}

\authorcomment3{Why are we commenting inline like this?}

\end{document}
```

produces this nice color free thing:

![Screen Shot 2019-12-02 at 10 35 00 PM](https://user-images.githubusercontent.com/234283/70018437-0eebec00-1554-11ea-8c52-15c5b93f566d.png)

**Comment by augustfly** [comment]
As a followup, I used a diff and it shows that these changes were entirely intentional. In other words the developer of v6.3 make all the colors black everywhere, removing color from v6.3. 

Until I hear otherwise I'm going to leave it as a bug and perhaps switch it to an out of date documentation (though I was not informed we were going back to black only edits). 

https://github.com/AASJournals/AASTeX60/commit/0fca6455c3df01a12ff28d1ae03c6d6dadb28868#diff-f19e74ff9590826e03ac09d2540a0351R5701

https://github.com/AASJournals/AASTeX60/commit/0fca6455c3df01a12ff28d1ae03c6d6dadb28868#diff-f19e74ff9590826e03ac09d2540a0351R5853

**Comment by gregschwarz** [comment]
Yes, this was intentional. Apparently people hated the red and blue colors of the other versions so we went back to black with the different font styles to differentiate the change versions.

**Comment by augustfly** [comment]
@gregschwarz Okay, so I will mark this as a "documentation" bug, and we will need to remove all the color related instructions from the docs so as to not confuse authors. Note also that some of the edits (e.g., edit3) only used color and since it is black now, will need to be deleted as a tool since it doesn't do anything.  I'll leave this issue open.

**Comment by claudiavr** [comment]
The "instructions to authors" should be corrected to reflect the present version os AASTeX.

**Comment by claudiavr** [comment]
The "instructions to authors" could also comment if the modifications from version 1 to version 2 of a manuscript should be kept highlighted in a possible version 3.

**Comment by augustfly** [comment]
The instructions to authors have been modified so that the `\edit#` and `\authorcomment#` instructions no longer reference color(s). 

However, the `\edit3` command is rendered useless in AASTeX v6.3. I would advise a further modification to the instructions dropping `\edit3` entirely as it does nothing to the text. 

If there were a third version then authors could simply update the text from version 2 to version 3 by deleting any `\edit1` commands from the text, changing any `\edit2` command to `\edit1` and adding new `\edit2` commands. 

This problem does not affect `\authorcomment3` because that command adds the plain text in parenthesis and prepends the "Author Note:" text.
